lib/notification_manager.rb

```diff
@@ -40,10 +40,6 @@ def cap_logger
     def sciencewire_logger
       @@sciencewire_logger ||= Logger.new(Settings.SCIENCEWIRE.LOG)
     end
-
-    def wos_logger
-      @@wos_logger ||= Logger.new(Settings.WOS.LOG)
-    end
     # rubocop:enable Style/ClassVars
```

Comment:
Merge this change with the addition of this method into the `WebOfScience` module, so it is clearer that this is a move-method refactor.